repository/templates/pages/page_detail.html

```diff
@@ -2,6 +2,19 @@
 
 {% block title %}{{ page.title }}{% endblock %}
 
+{% block social-meta-tags %}
+<meta property="og:site_name" content="{{settings.SITE_NAME}}"/>
+<meta property="og:title" content="{{settings.SITE_NAME}} - {{page.title}}"/>
```

Comment:
Being super-duper anal about this (also because @zarino loves typography), can these dashes be `&ndash;`?